application-services/custom/fledge-export/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/edgexfoundry/app-functions-sdk-go/appsdk"
+	"github.com/edgexfoundry/app-functions-sdk-go/pkg/transforms"
+
+	fledgeTransforms "fledge-export/pkg/transforms"
+)
+
+func main() {
+
+	// 1) First thing to do is to create an instance of the EdgeX SDK, giving it a service key
+	edgexSdk := &appsdk.AppFunctionsSDK{
+		ServiceKey: "FledgeExport", // Key used by Registry (Aka Consul)
+	}
+
+	// 2) Next, we need to initialize the SDK
+	if err := edgexSdk.Initialize(); err != nil {
+		message := fmt.Sprintf("SDK initialization failed: %v\n", err)
+		if edgexSdk.LoggingClient != nil {
+			edgexSdk.LoggingClient.Error(message)
+		} else {
+			fmt.Println(message)
+		}
+		os.Exit(-1)
+	}
+
+	// 3) Shows how to access the application's specific configuration settings.
+	fledgeEndpoint, err := edgexSdk.GetAppSettingStrings("FledgeSouthHTTPEndpoint")
+	if err != nil {
+		edgexSdk.LoggingClient.Error(err.Error())
+		os.Exit(-1)
```

Comment:
What is this syntax? `the . Exit ( - 1 )` => `os.Exit(-1)`

Reply:
it is os.Exit(-1).  Something else needed?